include/nbl/builtin/hlsl/math/quadrature/gauss_legendre/impl.hlsl

```diff
@@ -1,8 +1,8 @@
 // Copyright (C) 2018-2023 - DevSH Graphics Programming Sp. z O.O.
 // This file is part of the "Nabla Engine".
 // For conditions of distribution and use, see copyright notice in nabla.h
-#ifndef _NBL_BUILTIN_HLSL_MATH_QUADRATURE_GAUSS_LEGENDRE_INCLUDED_
-#define _NBL_BUILTIN_HLSL_MATH_QUADRATURE_GAUSS_LEGENDRE_INCLUDED_
+#ifndef _NBL_BUILTIN_HLSL_MATH_QUADRATURE_GAUSS_LEGENDRE_IMPL_INCLUDED_
+#define _NBL_BUILTIN_HLSL_MATH_QUADRATURE_GAUSS_LEGENDRE_IMPL_INCLUDED_
 
 #include <nbl/builtin/hlsl/cpp_compat.hlsl>
```

Comment:
X-macros are not supposed to have an include guard or include other files (they're just pretty copy-pastable blocks of code)